spyderlib/requirements.py

```diff
@@ -39,7 +39,7 @@ def check_qt():
     try:
         import qtpy
         package_name, required_ver = qt_infos[qtpy.API]
-        actual_ver = qtpy.__version__
+        actual_ver = qtpy.PYQT_VERSION

```

Comment:
Because this one is only about supporting the newest qtpy. Please prepare it and I'll merge it :-)